samples/synchronization/src/main.c

```diff
@@ -10,10 +10,10 @@
 #include <zephyr/sys/printk.h>
 
 /*
- * The hello world demo has two threads that utilize semaphores and sleeping
+ * The synchronization demo has two threads that utilize semaphores and sleeping
  * to take turns printing a greeting message at a controlled rate. The demo
- * shows both the static and dynamic approaches for spawning a thread; a real
- * world application would likely use the static approach for both threads.
+ * shows only the dynamic approach for spawning a thread. Alternatively, 
```

Comment:
I know this is confusing/awkward but could you please remove the trailing whitespace there? :)